File: GameStoreXamarin/GameStoreXamarin.Core/Models/Product.cs
using SQLite;

namespace GameStoreXamarin.Core.Models
{
    public class Product
    {
        [PrimaryKey, AutoIncrement]
        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public double Price { get; set; }
        public byte[] Image { get; set; }
    }
}

File: GameStoreAngular4/AngularUI/src/app/product.ts
export class Product {
    constructor(
        public id: number, 
        public productName: string, 
        public price: number, 
        public image: string) 
    { }
}
export class ResponseResult {
    constructor(
        public code: string, 
        public message: string) 
    { }
 }